packages/server/src/users/api/UserDto.ts

```diff
@@ -0,0 +1,55 @@
+import { ApiProperty, ApiPropertyOptional, OmitType } from '@nestjs/swagger';
+
+import { FavouriteRestaurantDto } from '../../restaurants/api/RestaurantDto';
+import { FavouriteDishDto } from '../../restaurants/dishes/api/DishDto';
+import { ApiObjectIdProperty } from '../../shared';
+
+interface Card {
+  number: string;
+  expirationDate: Date;
+  securityCode: string;
+}
+
+class UserDto {
+  @ApiObjectIdProperty()
+  readonly id: string;
+
+  @ApiPropertyOptional()
+  readonly name: string;
+
+  @ApiPropertyOptional()
+  readonly surname: string;
+
+  @ApiProperty()
+  readonly email: string;
+
+  @ApiProperty()
+  readonly password: string;
```

Comment:
Koniec końców chyba hasło można całkiem wywalić, bo po tym jak zamknął się issue z auth widzę że hasło idzie tam przez [osobne DTO](https://github.com/CodersCamp2021-HK/CodersCamp2021.Project.Fullstack/blob/main/packages/server/src/auth/api/RegisterAsUserDto.ts).

A skoro tak to UserDto nie jest wysyłany przy rejestracji, tylko jedynie jako informacja zwrotna dla usera, więc hasło raczej nie jest w ogóle potrzebne (DTO nie musi odpowiadać 1:1 bazie danych).